File: Algoritmos_Ordenamiento/burbuja.go
func bubbleSort(lista []int) {
	n := len(lista)
	for i := 0; i < n; i++ {
		for j := 0; j < n-i-1; j++ {
			if lista[j] > lista[j+1] {
				lista[j], lista[j+1] = lista[j+1], lista[j]
			}
		}
	}
}

File: Algoritmos_Ordenamiento/burbuja.dart
void bubbleSort(List<int> lista) {
  int n = lista.length;
  for (int i = 0; i < n; i++) {
    for (int j = 0; j < n - i - 1; j++) {
      if (lista[j] > lista[j + 1]) {
        int temp = lista[j];
        lista[j] = lista[j + 1];
        lista[j + 1] = temp;
      }
    }
  }
}